BigBaseV2/src/hooks/protections/receive_net_message.cpp

```diff
@@ -1,86 +1,60 @@
 #include "hooking.hpp"
 #include "services/players/player_service.hpp"
+#include <natives.hpp>
 
 namespace big
 {
-
-	inline bool get_message_type(rage::netConnection::MessageType& msg_type, rage::datBitBuffer& buffer)
+	bool getMessageType(rage::eNetMessage& msgType, rage::datBitBuffer& buffer)
 	{
 		uint32_t pos;
 		uint32_t magic;
 		uint32_t length;
 		uint32_t extended{};
 		if ((buffer.m_flagBits & 2) != 0 || (buffer.m_flagBits & 1) == 0 ? (pos = buffer.m_curBit) : (pos = buffer.m_maxBit),
-			buffer.m_bitsRead + 15 > pos || !buffer.ReadDword(&magic, 14) || magic != 0x3246 || !buffer.ReadDword(&extended, 1))
-		{
-			msg_type = rage::netConnection::MessageType::MsgInvalid;
+			buffer.m_bitsRead + 15 > pos || !buffer.ReadDword(&magic, 14) || magic != 0x3246 || !buffer.ReadDword(&extended, 1)) {
+			msgType = rage::eNetMessage::CMsgInvalid;
 			return false;
 		}
-
 		length = extended ? 16 : 8;
-
-		if ((buffer.m_flagBits & 1) == 0 ? (pos = buffer.m_curBit) : (pos = buffer.m_maxBit), length + buffer.m_bitsRead <= pos && buffer.ReadDword((uint32_t*)&msg_type, length))
+		if ((buffer.m_flagBits & 1) == 0 ? (pos = buffer.m_curBit) : (pos = buffer.m_maxBit), length + buffer.m_bitsRead <= pos && buffer.ReadDword((uint32_t*)&msgType, length))
 			return true;
 		else
 			return false;
 	}
-
 	bool hooks::receive_net_message(void* netConnectionManager, void* a2, rage::netConnection::InFrame* frame)
 	{
 		if (frame->get_type() == 4)
 		{
 			rage::datBitBuffer buffer((uint8_t*)frame->m_data, frame->m_length);
 			buffer.m_flagBits = 1;
-			rage::netConnection::MessageType msg_type;
-
+			rage::eNetMessage msgType;
 			const auto player = g_player_service->get_by_msg_id(frame->m_msg_id);
-			if (player && get_message_type(msg_type, buffer))
+			if (player && getMessageType(msgType, buffer))
```

Comment:
Why did you swap to camelCasing, the entire base uses snake_casing.

Reply:
forgor